tests/test_core.py

```diff
@@ -4221,14 +4221,7 @@ def test_strtod(self):
     self.do_run(src, expected)
 
   def test_strtold(self):
-    if not self.is_wasm_backend():
-      # XXX add real support for long double
-      expected_file = 'test_strtod.out'
-    else:
-      expected_file = 'test_strtold.out'
-    src = open(path_from_root('tests', 'core', 'test_strtold.c')).read()
-    expected = open(path_from_root('tests', 'core', expected_file)).read()
-    self.do_run(src, expected)
+    self.do_run_in_out_file_test('tests', 'core', 'test_strtold')
```

Comment:
Why this change?  Did fastcomp get fixed?  Maybe a separate change?

Reply:
This changed because the wasm backend used to print float128s with full precision, and after this PR, with float64 precision. There is no change to fastcomp - the wasm backend now emits the same as fastcomp here (so no more need to have two options for different outputs in this test).